MC/bin/o2dpg_sim_workflow_anchored.py

```diff
@@ -141,43 +129,25 @@ def retrieve_params_fromGRPECS(ccdbreader, run_number, rct = None):
 
     match_object=re.match("\s*([0-9]*)\s*-\s*([0-9]*)\s*.*", VALIDITY)
     SOV = -1  # start of object validity (not necessarily the same as actual run-start)
-    EOV = -1  # end of object validity (not the same as actual run-end)
     if match_object != None:
       SOV=match_object[1]
-      EOV=match_object[2]
 
     # we make a suitable request (at the start time) --> this gives the actual
     # object, with which we can query the end time as well
     grp=retrieve_CCDBObject_asJSON(ccdbreader, "/GLO/Config/GRPECS" + "/runNumber=" + str(run_number) + "/", int(SOV))
 
-
     # check that this object is really the one we wanted based on run-number
     assert(int(grp["mRun"]) == int(run_number))
 
-    SOR=int(grp["mTimeStart"]) # in milliseconds
-    EOR=int(grp["mTimeEnd"])
-    # cross check with RCT if available
-    if rct != None:
-       # verify that the variaous sor_eor information are the same
-       if SOR != rct["SOR"]:
-         print ("WARNING: Inconsistent SOR information on CCDB (divergence between GRPECS and RCT) ... will take RCT one")
-         SOR=rct["SOR"]
-
-       if EOR != rct["EOR"]:
-         print ("WARNING: Inconsistent EOR information on CCDB (divergence between GRPECS and RCT) ... will take RCT one")
-         EOR=rct["EOR"]
-
```

Comment:
I took all these things out. Now, simply the given `run_start` and `run_end` are used where needed.